### ISSUE TITLE
Tooltip title unable to be announced by screen reader when receiving keyboard focus.

### ISSUE THREAD

**Issue by ethib137** [liferay/clay#5189]
https://codesandbox.io/s/bold-jones-moit15?file=/src/index.js

Since we added the ability to show the tooltip on keyboard focus, we inadvertently removed the title attribute which the screen reader would read. This results in some elements that previously were announced correctly by a screen reader to now be announced as unlabeled.

If we know when the tooltip is being shown due to keyboard focus, we could NOT remove the title attribute in that instance, since we don't need to worry about the native title tooltip showing because the element is not being hovered. Alternatively, we could add an aria-label attribute set to the value of the title when the tooltip is shown. 

```
// Option A
if (!hovered) {
    // don't replace title with data-restore-title
}

// Option B
if (!element.hasAttribute('aria-label')) {
    element.setAttribute('aria-label', element.getAttribute('title'));
}

element.setAttribute('data-restore-title', element.getAttribute('title'));
element.removeAttribute('title')

// clean up
```

Would either of those options work? Any different ideas that might be better?

cc: @marcoscv-work

**Comment by ethib137** [comment]
Thanks @matuzalemsteles! Yep that was the approach I was leaning towards as well and for the same reasons. 😄
